singer_sdk/plugin_base.py

```diff
@@ -28,6 +28,7 @@
 from singer_sdk.exceptions import ConfigValidationError
 from singer_sdk.helpers._classproperty import classproperty
 from singer_sdk.helpers._compat import metadata
+from singer_sdk.helpers._meltano import meltano_yaml_str
```

Comment:
Is this a leftover from refactoring the other PR?

Reply:
Good catch - yes! Will remove...

Reply:
This whole file, actually.